packages/flutter/lib/src/widgets/slotted_render_object_widget.dart

```diff
@@ -0,0 +1,271 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+
+import 'framework.dart';
+
+/// A mixin for a [RenderObjectWidget] that configures a [RenderObject]
+/// subclass, which organizes its children in different slots.
+///
+/// Implementors of this mixin have to provide the list of available slots by
```

Comment:
"Implementor" => "Implementer"

I thought there was something about this in the style guide but I couldn't find it.  Anyway, grepping through the rest of the framework code I see that it's always spelled with "er".